build.gradle.kts

```diff
@@ -27,10 +27,77 @@ val collectCatalogTypings by tasks.registering(Sync::class) {
 
 val collectExternalTypings by tasks.registering(Sync::class) {
     mapOf(
-        "Vampire_setup-wsl_v4.yml" to "Vampire/setup-wsl/refs/heads/v4/action-types.yml",
-        "Vampire_setup-wsl_master.yml" to "Vampire/setup-wsl/refs/heads/master/action-types.yml",
-        "typesafegithub_github-actions-typing_v1.yml" to "typesafegithub/github-actions-typing/refs/heads/v1/action-types.yml",
+        "abhirajputnikku7_android-emulator-runner_main.yml" to "abhirajputnikku7/android-emulator-runner/refs/heads/main/action-types.yml",
```

Comment:
I'm wondering if we could specify the action coords just once per action, and generate both the part of the URL and the target file name? Now there's some duplication and impl details related to the URL and the file name.

Reply:
Maybe, but I didn't want to over-engineer it.
Currently the pattern is also the same almost everywhere, except that some have `yml`, some `yaml` and some `master`, some `main`.
But if in the future e. g. also repos with nested actions or multiple actions come in, then this has also to be considered accordingly and so on.
So I thought it might be easier to just have this little duplication.

As the names are relatively irrelevant, I could alternatively, also just list the URL and for the file-name replace all slashes by underscores.

Reply:
Ok, makes sense to keep things simple and explicit for now, to not build too much of a machinery.

One more question: how did you list these? Using the dependency graph manually ([1](https://github.com/typesafegithub/github-actions-typing/network/dependents), [2](https://github.com/typesafegithub/github-actions-typing/network/dependents?package_id=UGFja2FnZS0zNjg4NzA3NjYz)), or via some API? If API, could you share a request? Then we can use it in the build logic, or at least put it in the comment before this list for easy manual regeneration. For automatic iteration, one problem here might be forks - I think we don't want to test them, so if it's hard to exclude the forks, let's not bother with automatic iteration.

Reply:
Manually from that Insights page you linked to.
I'm not aware of an endpoint in the REST API with which you can get the dependents.
And even if you could, you would then still have to determine the type manifest files. (yaml vs. yml, path to sub-action, branch name, ...)